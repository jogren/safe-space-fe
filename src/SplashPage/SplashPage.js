import React from 'react'
import './SplashPage.css'

const SplashPage = () => {
  return (
    <section className="SplashPage">
      <img alt="lotus-flower" id="load-img-lotus" src={require('../assets/lotus-flower.svg')}/>
      <h1 id="loading-text">SafeSpace</h1>
    </section>
  )
}

export default SplashPage
