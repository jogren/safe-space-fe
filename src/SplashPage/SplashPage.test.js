import React from 'react';
import { shallow } from 'enzyme';
import { SplashPage } from './SplashPage';


describe('SplashPage', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SplashPage />)
  })

  it('should match the snapshot with the data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  })
})