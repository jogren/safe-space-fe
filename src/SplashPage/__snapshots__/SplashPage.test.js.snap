// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplashPage should match the snapshot with the data passed through 1`] = `
<section
  className="SplashPage"
>
  <img
    alt="lotus-flower"
    id="load-img-lotus"
    src={
      Object {
        "ReactComponent": Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        },
        "__esModule": true,
        "default": "lotus-flower.svg",
      }
    }
  />
  <h1
    id="loading-text"
  >
    SafeSpace
  </h1>
</section>
`;
